generators/app/templates/graphQL/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.14.1
```

Comment:
same, i think this should be the same node file

Reply:
ok!